main/cicd/cicd-pipeline/config/infra/cloudformation.yml

```diff
@@ -228,6 +231,20 @@ Resources:
       PolicyDocument:
         Version: '2012-10-17'
         Statement:
+          - Action:
+              - 'route53:GetChange'
+              - 'route53:ListHostedZones'
+              - 'route53:CreateHostedZone'
```

Comment:
which step of the pipeline is creating hosted zones?

Reply:
I think when it deploys backend stack, in one of the step it complains about not having permission for it. I can check by removing it and deploying it.

Reply:
We will test it while creating ci/cd prod, for this PR leaving it as it is.